feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/post/PostItem.kt

```diff
@@ -87,6 +91,7 @@ internal fun PostItem(
     modifier: Modifier,
     post: IBasePost?,
     postItemViewType: PostItemViewType,
+    profilePictureImageProvider: ProfilePictureImageProvider?,
```

Comment:
This handing through of the profilePictureImageProvider seems rather tedious. I think we should use a composition local for this.